Modules/Tokens/src/Tokens.Application/Tokens/Commands/CreateToken/Handler.cs

```diff
@@ -21,7 +21,9 @@ public Handler(IUserContext userContext, IMapper mapper, ITokensRepository token
 
     public async Task<CreateTokenResponse> Handle(CreateTokenCommand request, CancellationToken cancellationToken)
     {
-        var newTokenEntity = new Token(_userContext.GetAddress(), _userContext.GetDeviceId(), request.Content, request.ExpiresAt);
+        var newTokenEntity = request.ForIdentity == null ?
+            new Token(_userContext.GetAddress(), _userContext.GetDeviceId(), request.Content, request.ExpiresAt, null) :
+            new Token(_userContext.GetAddress(), _userContext.GetDeviceId(), request.Content, request.ExpiresAt, request.ForIdentity);
```

Comment:
I don't get it. Why not just

```suggestion
        var newTokenEntity = new Token(_userContext.GetAddress(), _userContext.GetDeviceId(), request.Content, request.ExpiresAt, request.ForIdentity);
```

?